keyboards/akko/top40/keymaps/vial/config.h

```diff
@@ -0,0 +1,20 @@
+// Copyright 2023 Oleg Titov <oleg.titov@gmail.com>
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+// Unique keyboard id
+#define VIAL_KEYBOARD_UID {0x5D, 0x72, 0x7C, 0xB4, 0xC9, 0x7C, 0xF2, 0x60}
+
+// Unlock with Escape+Enter
+#define VIAL_UNLOCK_COMBO_ROWS { 0, 1 }
+#define VIAL_UNLOCK_COMBO_COLS { 0, 11 }
+
+// Enable all available RGB Matrix effects
+#define RGB_MATRIX_FRAMEBUFFER_EFFECTS
+#define RGB_MATRIX_KEYPRESSES
```

Comment:
```suggestion

```
Why are you reintroducing these rules when 1) These are already defined [at the keyboard level](https://github.com/qmk/qmk_firmware/blob/master/keyboards/akko/top40/config.h#L48-L50) 2) Enabling [`VIALRGB_ENABLE = yes` in the `vial` keymap](https://github.com/vial-kb/vial-qmk/blob/a9f7750316f5e3230c1c3739f7f74b520a0b61ae/keyboards/akko/top40/keymaps/vial/rules.mk#L3) will would actually make every effect be enabled?

Reply:
Mea culpa. I have not checked keyboards level config at first. I had no knowledge about relationship between above mentioned `#define`'s and `VIALRGB_ENABLE = yes`, thank you. Give me a second to fix this.